src/components/node/stanford-person/page-display.tsx

```diff
@@ -110,64 +90,67 @@ const StanfordPerson = async ({node, ...props}: {node: NodeStanfordPerson}) => {
+          {(node.suPersonTelephone ||
+            node.suPersonMobilePhone ||
+            node.suPersonFax ||
+            node.suPersonMailCode ||
+            node.suPersonEmail) && (
+            <div>
+              <h2 className="text-24">Contact</h2>
+              <ul className="list-none p-0 children:mb-[0.4rem] last:children:mb-0">
+                {node.suPersonTelephone && (
+                  <li>
+                    <PhoneIcon title="Phone" width={24} className="mr-4 inline-block text-digital-blue" />p{" "}
+                    <NumberLink tel={node.suPersonTelephone} />
+                  </li>
+                )}
+                {node.suPersonMobilePhone && (
+                  <li>
+                    <PhoneIcon title="Phone" width={24} className="mr-4 inline-block text-digital-blue" /> m{" "}
+                    <NumberLink tel={node.suPersonMobilePhone} />
+                  </li>
+                )}
+                {node.suPersonFax && (
+                  <li>
+                    <PhoneIcon title="Phone" width={24} className="mr-4 inline-block text-digital-blue" />f{" "}
+                    <NumberLink tel={node.suPersonFax} />
+                  </li>
+                )}
+                {node.suPersonEmail && (
+                  <li>
+                    <EnvelopeIcon title="Email" width={20} className="mr-4 inline-block text-digital-blue" />
+                    <EmailLink className="break-words" email={node.suPersonEmail} />
+                  </li>
+                )}
+                {node.suPersonMailCode && <li>Mail Code: {node.suPersonMailCode}</li>}
+              </ul>
+            </div>
+          )}
 
-                {node.suPersonLocationName && <div>{node.suPersonLocationName}</div>}
+          {(node.suPersonLocationName || node.suPersonLocationAddress || node.suPersonMapUrl) && (
+            <div className="children:mb-[0.4rem] last:children:mb-0">
+              <h2 className="text-24">Location</h2>
 
-                {node.suPersonLocationAddress && (
-                  <div className="children:mb-0">{formatHtml(node.suPersonLocationAddress.processed)}</div>
-                )}
+              {node.suPersonLocationName && <div>{node.suPersonLocationName}</div>}
 
-                {node.suPersonMapUrl?.url && (
-                  <div>
-                    Map URL: <Link href={node.suPersonMapUrl.url}>{node.suPersonMapUrl.url}</Link>
-                  </div>
-                )}
-              </>
-            )}
-          </div>
+              {node.suPersonLocationAddress && (
+                <div className="children:mb-0">{formatHtml(node.suPersonLocationAddress.processed)}</div>
+              )}
+
+              {node.suPersonMapUrl?.url && (
+                <div>
+                  <MapIcon title="Location" width={26} className="mr-4 inline-block text-digital-blue" />
```

Comment:
I would avoid using `inline-block`. I find it can cause unexpected results. Might think about using `flex items-center` on the parent to make the icon and the link text vertically centered. That goes for the other icon elements as well.

Reply:
Thank you Mike! I'll get right on this fixup 🫡